### PR TITLE
Fix integration test

### PR DESCRIPTION
**Pull request by minsulee2**
Updates here:
- Remove sleep package
- Use Chainutil.sleep() instead
- Have tested in act environment!

### CODE REVIEW

File: .github/workflows/local/github-actions-for-local.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+          registry-url: 'https://registry.npmjs.org'
+      - name: Install Yarn
+        run: sudo npm install -g yarn
+      - name: install pip
+        run: |
+          sudo apt-get install software-properties-common
+          sudo apt-add-repository universe
+          sudo apt-get update
+          sudo apt-get -y install python-pip
+          pip install ast
```
Comment (liayoo) [inline .github/workflows/local/github-actions-for-local.yml:33]: question) why do we need pip and ast?

File: unittest/test-util.js [hunk 2]
```diff
@@ -81,39 +80,39 @@ function waitUntilTxFinalized(servers, txHash) {
...
           .body
           .toString('utf-8')).result;
       if (txStatus && txStatus.is_finalized === true) {
         unchecked.delete(server);
       }
-    });
-    sleep(200);
-    iterCount++;
+      await ChainUtil.sleep(200);
+      iterCount++;
```
Comment (platfowner) [inline unittest/test-util.js:91]: Need to be moved out of this block?

File: unittest/test-util.js [hunk 3]
```diff
@@ -81,39 +80,39 @@ function waitUntilTxFinalized(servers, txHash) {
...
       const txStatus = parseOrLog(syncRequest('GET', server + `/get_transaction?hash=${txHash}`)
           .body
           .toString('utf-8')).result;
       if (txStatus && txStatus.is_finalized === true) {
         unchecked.delete(server);
       }
-    });
-    sleep(200);
-    iterCount++;
+      await ChainUtil.sleep(200);
```
Comment (platfowner) [inline unittest/test-util.js:90]: Question: Is this change necessary for fixing the test?
Just curious as sleep() is left as-is in the previous refactoring since it makes difference only in performance, which is not that critical in tests?

File: .github/workflows/local/github-actions-for-local.yml (hunk 1 above)
Comment (minsulee2) [reply .github/workflows/local/github-actions-for-local.yml:33]: In the act environment, it is not exactly the same as the github actions environment. Since the setup-node@v2 image in act app haven't got ast module in python pip by default, the yarn install cannot be run. To fix this, pip install ast has been added!

File: unittest/test-util.js (hunk 3 above)
Comment (minsulee2) [reply unittest/test-util.js:90]: The sleep package does not work in linux environment at all. So I removed them all in the test codes.

File: unittest/test-util.js (hunk 2 above)
Comment (minsulee2) [reply unittest/test-util.js:91]: Can you explain it more?

File: unittest/test-util.js (hunk 3 above)
Comment (platfowner) [reply unittest/test-util.js:90]: I see. Thanks for the explanation.

Comment (platfowner) [reply unittest/test-util.js:90]: As you can see, the two lines used to be out of the forEach() block and for the outer while loop.

File: unittest/test-util.js (hunk 2 above)
Comment (platfowner) [reply unittest/test-util.js:91]: The two lines (sleep() + iterCount++) used to be out of the forEach() loop (now just for-loop) sleeping for and counting the iteration of the outer while-loop.

Comment (minsulee2) [reply unittest/test-util.js:91]: Oh, I removed .forEach() cuz the await does't work in the .forEach(). Now the two lines are in for() loop. Please check the code above :)

Comment (platfowner) [reply unittest/test-util.js:91]: I mean.. sleep() for example doesn't need to be called for each iteration of the inner loop (the server checks are independent of each other). It was intended to have some waiting time for the outer while-loop (not for inner loop). iterCount was also intended to count the iteration of the while-loop (not inner loop).

Comment (minsulee2) [reply unittest/test-util.js]: My apology! I got confused the lines.. Fixed!!

**Review by platfowner — APPROVED** [review]
LGTM
Thanks for handling this issue!

File: .github/workflows/local/github-actions-for-local.yml (hunk 1 above)
Comment (liayoo) [reply .github/workflows/local/github-actions-for-local.yml:33]: 👌 Thanks for the explanation!

**Comment by liayoo** [comment]
LGTM, thanks!

**Review by liayoo — APPROVED** [review]

**Comment by minsulee2** [comment]
Thanks for the reviews!
